src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.AnyUnix.cs

```diff
@@ -0,0 +1,1793 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why does it need to be renamed?

Reply:
There are some functions in the original `TimeZoneInfo.Unix.cs` that will not work for Android, so those need to be split out. The files would be `(Used by all)` `(Used by non-Android)` `(Used by Android)`, where all refers to `TargetsUnix and TargetsBrowser`. What would be the preferred name for the file that contains the `non-Android` implementation (and the other files)?

Reply:
AnyUnix was the only thing that came to mind.  I couldn't find a similar implementation to suggest anything else.